plugins/modules/event.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# Copyright: (c) 2019, Paul Arthur <paul.arthur@flowerysong.com>
+# Copyright: (c) 2019, XLAB Steampunk <steampunk@xlab.si>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'XLAB Steampunk'}
+
+DOCUMENTATION = '''
+module: event
+author:
+  - Paul Arthur (@flowerysong)
+  - Aljaz Kosir (@aljazkosir)
+  - Miha Plesko (@miha-plesko)
+  - Tadej Borovsak (@tadeboro)
```

Comment:
I wrote zero lines of code in this module and do not deserve to be on this list, so feel free to remove me from it.

Reply:
Originally we wanted to use
```
- Steampunk
```
in there, but I think sanity complains unless you specify a valid GH handle next to it. Then we decided to just list the three related engineers instead. Maybe we should create a GH user named "steampunk" to handle situations like this?

Reply:
I would avoid creating a new GitHub user just for this since someone would need to monitor the messages on that artificial thing.